internal/usage/usage.go

```diff
@@ -393,4 +393,6 @@ dbName and collection are required only for built-in roles.`
 	OperatorTargetNamespace                   = "Namespaces to use for generated kubernetes entities"
 	OperatorVersion                           = "Version of Atlas Kubernetes Operator to generate resources for."
 	ExportID                                  = "Unique string that identifies the AWS S3 bucket to which you export your snapshots."
+	RequiredRoleIntro                         = "To use this command, you must authenticate with a user account or an API key that has the"
+	RequiredRoleClose                         = "role."
```

Comment:
you could combine them like
```suggestion
	RequiredRole                         = "To use this command, you must authenticate with a user account or an API key that has the %s role."
```

then use it like
```go
Long: fmt.Sprintf(usage.RequiredRoleIntro, "Project Read Only"),
```